runtime/rasdump/javadump.cpp

```diff
@@ -4313,7 +4313,11 @@ JavaCoreDumpWriter::writeMonitorObject(J9ThreadMonitor* monitor, j9object_t obj,
 	if (NULL != obj) {
 		owner = getObjectMonitorOwner(_VirtualMachine, obj, &count);
 	} else if (NULL != lockOwner) {
-		owner = getVMThreadFromOMRThread(_VirtualMachine, lockOwner);
+		if (IS_J9_OBJECT_MONITOR_OWNER_DETACHED(lockOwner)) {
+			owner = NULL;
```

Comment:
owner is already initialized to null, should be able to just check the else case.